src/Writer/MSSQL.php

```diff
@@ -136,12 +133,7 @@ public function write(CsvFile $csv, array $table)
         $this->logger->info("BCP import finished");
     }
 
-    public function isTableValid(array $table, $ignoreExport = false)
```

Comment:
koukal jsem, že to není součást interfacu ani ničeho a je to tam od začátku a v kódu jsem o tom nenašel (krom tady) zmínku, ale nedokážu asi vyzkoumat, jestli se to někde nemůže používat - je to public API.

Reply:
Je to taka legacy vec, ktora bola sucastou interfaceu, ale uz som ju vyhodil, takze ju vyhodim aj odtialto. Toto nie je knihovna, takze pokial sa nepouziva v kode tohto projektu, tak to myslim nicomu nevadi, ked sa to rovno zmaze :)